src/main/java/org/example/CurrencySymbolCheck.java

```diff
@@ -0,0 +1,84 @@
+package org.example;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class CurrencySymbolCheck {
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+    private final By checkCurrency = By.xpath("//*[@class='main-goods__currency']");
+    private final By searchField = By.xpath("//*[@name=\"search\"]");
+    private final By searchButton = By.xpath("//*[@role=\"search\"]/form/button");
+    private final By buyProductButton = By.xpath("//*[@class=\"product__buy\"]/*/button");
+    //private final By checkCurrency = By.xpath("//*[@class='main-goods__currency']");
+    private final By closePopupButton = By.xpath("//*[@class ='modal__header']/button");
+    private final By productSeller = By.className("product-seller__body");
+    private final By openCart = By.xpath("//*[@class='js-rz-cart']/div[1]/a");
+    private final By plusButton = By.xpath("//*[@class='cart-product__footer']//div/button[2]");
+    private final int TIME_OUT_IN_SECONDS = 30;
+
+    public CurrencySymbolCheck(WebDriver driver) {
```

Comment:
Page objects should be named as a page or part of the page which they represent, i.e. SearchResultPage, CartPopup